bigml/bigmlconnection.py

```diff
@@ -252,10 +258,42 @@ def _create(self, url, body, verify=None):
             verify = self.verify
 
         while code == HTTP_ACCEPTED:
+            if GAE_ENABLED:
+                try:
+                    req_options = {
+                        'url': url + self.auth,
+                        'method': urlfetch.POST,
+                        'headers': SEND_JSON,
+                        'payload': body,
+                        'validate_certificate': verify
+                    }
+                    response = urlfetch.fetch(**req_options)
+                except urlfetch.Error, exception:
+                    LOGGER.error("Error establishing connection: %s",
+                                 str(exception))
+                    return maybe_save(resource_id, self.storage, code,
+                                      location, resource, error)
+            else:
+                try:
+                    response = requests.post(url + self.auth,
+                                             headers=SEND_JSON,
+                                             data=body, verify=verify)
+                except requests.ConnectionError, exc:
+                    LOGGER.error("Connection error: %s", str(exc))
+                    code = HTTP_INTERNAL_SERVER_ERROR
+                    return maybe_save(resource_id, self.storage, code,
+                                      location, resource, error)
+                except requests.Timeout:
+                    LOGGER.error("Request timed out")
+                    code = HTTP_INTERNAL_SERVER_ERROR
+                    return maybe_save(resource_id, self.storage, code,
+                                      location, resource, error)
+                except requests.RequestException:
+                    LOGGER.error("Ambiguous exception occurred")
+                    code = HTTP_INTERNAL_SERVER_ERROR
+                    return maybe_save(resource_id, self.storage, code,
+                                      location, resource, error)
```

Comment:
i think it's not worth triplicating essentially the same handler just to tweak a little the error message... why not just:

```python
try:
    ....
except  requests.RequestException, exc:
  Logger.error("HTTP request error: %s", str(exc))
  code =  HTTP_INTERNAL_SERVER_ERROR
  return maybe_save(...)
```
 and similarly for the other two or five cases below? we save lots of duplicate code and lose nothing, no?